src/front/ticketer/lib/pages/page_home.dart

```diff
@@ -17,13 +17,13 @@ class HomePage extends StatefulWidget {
 class _HomePageState extends State<HomePage> {
   @override
   Widget build(BuildContext context) {
-    User? user = Auth().getCurrentUser;
+    Account? user = Auth().getCurrentAccount;
 
     switch (user!.type) {
```

Comment:
This is very wrong but okay I'm going to let it slide for now. Regardless it needs rewriting.